titus-common/src/main/java/com/netflix/titus/common/util/cache/internal/InstrumentedCache.java

```diff
@@ -79,7 +79,7 @@ public InstrumentedCache(String metricNameRoot, Cache<K, V> cache, Registry regi
                 next -> {
                     if (next.isPresent()) {
                         Throwable cause = next.get();
-                        logger.error("Unable to update cache metrics with error: {}", cause);
+                        logger.error("Unable to update cache metrics with error: {}", String.valueOf(cause));
```

Comment:
Can you remove ` {}` and just put the cause as an argument instead of doing variable substitution?

Reply:
Done